staging/src/k8s.io/legacy-cloud-providers/gce/gce_util_test.go

```diff
@@ -118,3 +119,38 @@ func TestFirewallToGcloudArgs(t *testing.T) {
 		t.Errorf("%q does not equal %q", got, e)
 	}
 }
+
+// TestFinalizerUtils tests the add/remove and hasFinalizer methods.
+func TestFinalizerUtils(t *testing.T) {
```

Comment:
Usually don't name tests after files or packages but the functions you are testing:

TestFinalizerUtils

Reply:
Done